src/spec-configuration/httpOCIRegistry.ts

```diff
@@ -186,6 +194,20 @@ async function getCredential(params: CommonParams, ociRef: OCIRef | OCICollectio
 							refreshToken: undefined,
 						};
 					}
+					if (dockerConfig.credHelpers && dockerConfig.credHelpers[registry]) {
```

Comment:
Is there documentation on whether auths or credHelpers should take precedence? I also tried finding Docker's implementation, but somehow couldn't find it.